tests/test_connection.py

```diff
@@ -1005,6 +1005,27 @@ def test_funding_external_wallet_corners(node_factory, bitcoind):
     # We can cancel channel after fundchannel_complete
     assert l1.rpc.fundchannel_cancel(l2.info['id'])['cancelled']
 
+    l1.rpc.connect(l2.info['id'], 'localhost', l2.port)
+    l1.rpc.fundchannel_start(l2.info['id'], amount)['funding_address']
+    assert l1.rpc.fundchannel_complete(l2.info['id'], prep['txid'], txout)['commitments_secured']
```

Comment:
This could do with a short description of what it tests. I guess i checks that we can still start and complete a fundchannel after cancelling?

Reply:
this is a bit of an API assertion, in that it checks that the field `commitments_secured` is returned and is set to true. if this API changes, this test will fail.